### PR TITLE
Greek Updated strings.po

### PR DESCRIPTION
**Pull request by Filosofos**
Hope i have only commit the changes in the right way now.

### CODE REVIEW

File: resources/language/Greek/strings.po [hunk 1]
```diff
@@ -148,7 +180,8 @@ msgstr "Απαιτείται η επανεκκίνηση του Kodi πριν α
 
 msgctxt "#30102"
 msgid "This add-on can only be run from within Quasar"
-msgstr "Το συγκεκριμένο πρόσθετο μπορεί να χρησιμοποιηθεί μόνο μέσα από το Quasar"
+msgstr ""
+"Το συγκεκριμένο πρόσθετο μπορεί να χρησιμοποιηθεί μόνο μέσα από το Quasar"
```
Comment (scakemyer) [inline resources/language/Greek/strings.po]: This doesn't seem right

**Comment by scakemyer** [comment]
Please mention the language in the commit message, not just `Update strings.po`.

**Comment by ditheo** [comment]
Dear Filosofos, i didn;t see your updated strings.po, so i requested a new pull. Sorry for that. But how we can work together, if you want it too? If you don't want to do the transaltion together can you make a change in your translation? The plugin seems that use one string for "most popular" both for movies and tv shows, and if anybody navigates to tv shows reads "δημοφιλέστερες ταινίες" instead "δημοφιλέστερες σειρές". To avoid this, in my translation i changed it to "δημοφιλέστερα"(most popular)  
[strings-ditheo.po.txt](https://github.com/scakemyer/plugin.video.quasar/files/130724/strings-ditheo.po.txt)

**Comment by Filosofos** [comment]
I am all new to github, so i try to update with edits. I think that the best way is to fork my repository (which i forked from scakemyer's etc) and to commit your changes to it (?). If scakemyer could give us a quick hint it would help us a lot. For start, give it a try like that. Thank you

**Comment by scakemyer** [comment]
`git rebase -i` into one commit please

**Comment by scakemyer** [comment]
Merged in c31ff31

**Pull request closed without merging**